### ISSUE TITLE
Implement new ntime for jobs

### ISSUE THREAD

**Issue by Fi3** [stratum-mining/stratum#353]
https://github.com/stratum-mining/sv2-spec/pull/12

**Comment by pavlenex** [comment]
@Fi3 is this a duplicate of https://github.com/stratum-mining/stratum/issues/335?

**Comment by Fi3** [comment]
yes close this one and assign the other one also to me so I don't forget that exist
